### PR TITLE
feat: added kusk configuration

### PR DESCRIPTION
**Pull request by jasmingacic**
Signed-off-by: Jasmin Gacic <jasmin.gacic@gmail.com>

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by fog1985 — APPROVED** [review]
LGTM

**Review by exu — APPROVED** [review]

File: api/v1/kusk.yaml [hunk 1]
```diff
@@ -0,0 +1,1147 @@
...
+                application/problem+json:
+                  schema:
+                    items:
+                      $ref: '#/components/schemas/Problem'
+                    type: array
+              description: problem with getting script executions from storage
+          summary: Get all script executions
+          tags:
+          - executions
+          - api
```
Comment (Tarick) [inline api/v1/kusk.yaml:573]: Another entry for dashboard to work (it prepends / to the api path), as we need to add /executions/ route:
```
/executions/:
        get:
          description: Returns array of scripts executions
          operationId: listExecutionsA
          parameters:
          - $ref: '#/components/parameters/ExecutionsPageSize'
          - $ref: '#/components/parameters/ExecutionsPageIndex'
          - $ref: '#/components/parameters/ExecutionsStatusFilter'
          - $ref: '#/components/parameters/ExecutionsStartDateFilter'
          - $ref: '#/components/parameters/ExecutionsEndDateFilter'
          responses:
            "200":
              content:
                application/json:
                  schema:
                    $ref: '#/components/schemas/ExecutionsResult'
              description: successful operation
            "500":
              content:
                application/problem+json:
                  schema:
                    items:
                      $ref: '#/components/schemas/Problem'
                    type: array
              description: problem with getting script executions from storage
          summary: Get all script executions
          tags:
          - executions
          - api
```

**Review by Tarick — CHANGES_REQUESTED** [review]
see comment on /executions/ entry

File: api/v1/kusk.yaml (hunk 1 above)
Comment (jasmingacic) [reply api/v1/kusk.yaml:573]: @exu do you have the file you used to do this? if so can you add the change so i don't break this

Comment (jasmingacic) [reply api/v1/kusk.yaml:573]: @Tarick I changed it can you please take another look

File: api/v1/kusk.yaml [hunk 2]
```diff
@@ -0,0 +1,1176 @@
...
+      /executions/:
```
Comment (jasmingacic) [inline api/v1/kusk.yaml:574]: From line 574 - 602

**Review by Tarick — APPROVED** [review]

**Pull request merged**